### PR TITLE
Override apache default modules and adjust vhost.

### PR DESCRIPTION
**Pull request by lliss**
I wanted indexes simply removed from all the servers. However there is
no easy way to remove an enabled module. The best we can do is prevent
it from installing initially. Further, I don't want the default vhosts
file to come in with a prepended 15 since we are defaulting this to be
named `000-default` anyway to get it to the top of the priority list.

### CODE REVIEW

File: manifests/server/apache.pp [hunk 1]
```diff
@@ -47,17 +25,34 @@
     purge_configs  => $purge_configs,
   }
 
+  # TODO Audit this list. Drupal actually doesn't need most of these.
```
Comment (tizzo) [inline manifests/server/apache.pp:28]: This list actually came from acquia - this was what they were installing at the time. The list was acquired for probo's environment building which was what lead to the creation of this module in the first place.

Comment (lliss) [inline manifests/server/apache.pp:28]: Got it. I've tried to find a comprehensive list of what is actually needed and ended up with a much smaller list but this seems okay to me. They had autoindex in that list. I wonder if that was for a reason.

Comment (lliss) [inline manifests/server/apache.pp:28]: Incidentally, that list was never actually being used by the module.

**Comment by tizzo** [comment]
Well, indexes are probably a thing that this module should be agnostic about but Drupal doesn't require it so neither should this module.

**Pull request merged**